packages/frontend/src/components/composer/AppPickerWrapper.tsx

```diff
@@ -52,17 +52,13 @@ export const AppPickerWrapper = ({ onAppSelected, apps, setApps }: Props) => {
           app.author = url.pathname.split('/')[1]
           app.date = moment(app.date).format('LL')
         }
-        if (apps) {
-          setApps(apps)
-        }
+        setApps(apps)
       } catch (error) {
         log.error('Failed to fetch apps:', error)
       }
     }
-    if (!apps?.length) {
-      fetchApps()
-    }
-  }, [apps, setApps])
+    fetchApps()
+  }, [setApps])
```

Comment:
Don't cache on `apps`, this is the result we want to display. The results are cached in core already.

Reply:
Well the original idea was not to reload all apps any time you (re)open the AppPicker, since changes are rare. But then we would at least have to set a client-side cache-expire time. If core cares about that (only reloading the apps if they changed) we don't need to care about that on the client. (which I didn't know when implementing this tbh)

Reply:
> so adding a second caching layer just makes things needlessly complicated

yes that's why I said above we don't need the client cache